packages/round-manager/src/features/api/services/program.ts

```diff
@@ -109,11 +113,14 @@ export const programApi = api.injectEndpoints({
               }
             `,
             chainId,
-            { address: address.toLowerCase() }
+            { address: address?.toLowerCase(), programId }
           )
 
           const programs: Program[] = []
 
+          console.log(res);
```

Comment:
nitpick: remove console.log

Reply:
resolved in [5d0721b](https://github.com/gitcoinco/grants-round/pull/297/commits/5d0721b02753d0ce9f240ec00b496ce21d4fc56c)